gateway-provider-security-jwt/src/test/java/org/apache/knox/gateway/provider/federation/JWTFederationFilterTest.java

```diff
@@ -43,6 +43,10 @@
 
 @SuppressWarnings("PMD.TestClassWithoutTestCases")
 public class JWTFederationFilterTest extends AbstractJWTFilterTest {
+
+  private static final String BASIC_ = "Basic ";
+  private static final String BEARER_ = "Bearer ";
```

Comment:
Is there any reason why `JWTFederationFilter.BEARER` and `JWTFederationFilter.BASIC` cannot be reused? I see we have an extra space after "Basic", but the "Bearer " seems to be the same.
Out of the scope of this PR, but we should consider introduce common constant interfaces (the `BEARER` constant is listed in at least 4 classes as `private`).

Reply:
Good call - let me fix that!